sycl/CMakeLists.txt

```diff
@@ -26,6 +26,10 @@ if(SYCL_ENABLE_WERROR)
   endif()
 endif()
 
+# Create a soft option for enabling or disabling the instrumentation 
+# of the SYCL runtime
+option(SYCL_ENABLE_XPTI_TRACING "Enable tracing of SYCL constructs" ON)
```

Comment:
Suggest disabling it by default and enabling through cmake parameters.

Reply:
@romanovvlad  Where does one enable this from the command line? In the cmake_cmd in configure.py ?

Reply:
What is cmake_cmd ?
AFAIK configure.py is used by CI only, developers just run cmake with options.

Reply:
@tovinkere users get the command line from get started guide. configure.py is only used by the buildbot. BTW, it would be great if you added support for xpti to confugure.py to enable testing later.

Reply:
@romanovvlad  The cmake_cmd line in buildbot/configure.py builds the command line for running CI and I have explicitly enabled XPTI tracing here.
@alexbatashev Currently, the sycl library links against a proxy/stub library and I'll work with you to add tests that make sense. When the XPTI framework (the implementation of the proxy library) is checked in as a reference implementation, a full-blown test suite will accompany it.